content/authors/Femi-ige-Mobolaji/index.md

```diff
@@ -0,0 +1 @@
+Femi-ige Mobolaji is a growing tech enthusiast trying to connect with a growing community. He is an entrepreneurship major from the Federal University of Minna.
```

Comment:
Please edit your author bio in the proper Markdown format. Here's an [example](https://raw.githubusercontent.com/section-engineering-education/engineering-education/master/content/authors/adith-bharadwaj/index.md)

Also, add your avatar i.e (author image).